server.py

```diff
@@ -210,6 +210,18 @@ def get_metrics():
         media_type="text/plain",
         content=prometheus_client.generate_latest(),
     )
+
+@app.get('/debug')
+def debug():
+    cache = Cache()
```

Comment:
can we use the global cache variable at the top of this file instead

Reply:
done